redDwarf03/archethic_mobile_wallet
# Contacts management enhanced

# Description

Contacts enhancement

Fixes #796 

## Type of change

- New feature (non-breaking change which adds functionality)

## Material used:

Please delete options that are not relevant.
- iOS (Smartphone/Tablet) : iPhone 14 Pro Max

## How Has This Been Tested?

Go into the contacts tab and see directly the contacts balance. Possibility to hide the balance on the top right, like the other screens.
In the detail of a contact, we can now :
- See his balance
- Create a new discussion
- New design : Remove the contact, add it to the favorites, or view the block chain info from the explorer

@redDwarf03 I didn't make the second point (swippable item for the list), because 1. the task was taking me too damn time but also because I think that this is not the right thing to do : for example in the contact app on iOS you cannot delete a contact otherwise than go into his detail (no swipe possible from the contact home screen)

### Patrol

Please list here the tests created in Patrol for this pull request.

## Checklist:

- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
- Any dependent changes have been merged and published in downstream modules

## Useful info for the reviewer:
